models/MaxCapacityResponse.v1.yaml

```diff
@@ -0,0 +1,29 @@
+title: MaxCapacityResponse
+type: object
+properties:
+  signins_count:
+    type: integer
+    description: The number of people currently present in the location
+  invites_expected_next_hour:
+    type: integer
+    description: The number of people expected in the next 60 mins from the current time.
+  max_capacity:
+    type: integer
+    description: The maximum capacity of a location.
+  chart_data:
+    type: array
+    description: Expected people in the next 6 hours
+    items:
+      type: integer
```

Comment:
This seems like and endpoint _specifically_ designed to power a single page in the mobile app. For a public-facing API, it needs to have other use cases in mind. Chart data, for example, need the X and Y values

Reply:
ok, so we use next n hours as an input here? So I assume it should be an integer. Like the next N number of hours?

Reply:
it will likely have to be a structure like this:

```json
[
  {
    "timestamp": "2020-07-17T17:00:00+00:00",  // ISO8601 timestamp
    "expected_visitors": 2
  },
  {
    "timestamp": "2020-07-17T18:00:00+00:00",  // ISO8601 timestamp
    "expected_visitors": 3
  }  
]
```

Reply:
We'll probably also need to allow the user to set a timezone as the start, in case they want to see future capacity. EG, a week from now, or tomorrow